### PR TITLE
docs: update 0rbit get and post request

### PR DESCRIPTION
**Pull request by Not-Sarthak**

### CODE REVIEW

**Comment by Not-Sarthak** [comment]
Add 0rbit guide for GET and POST requests

- Added documentation for handling GET and POST requests using 0rbit.
- Included code examples and usage instructions.
- Added it under the guides section inside `locales.js`

File: src/guides/0rbit/post-request.md [hunk 1]
```diff
@@ -1,96 +1,192 @@
...
-   Next, create your instance of aos:
+### Initialize the Variables
 
-```bash
-aos
+```lua
+local json = require("json")
+
+_0RBIT = "BaMK1dfayo75s3q1ow6AO64UDpD9SEFbeE8xYrY2fyQ"
+_0RBT_TOKEN = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
```
Comment (megabyte0x) [inline src/guides/0rbit/post-request.md:38]: @Not-Sarthak please change this to `_0RBT_POINTS`

File: src/guides/0rbit/get-request.md [hunk 2]
```diff
@@ -3,75 +3,159 @@ prev:
...
-   Next, create your instance of aos:
+### Initialize the Variables
 
-```bash
-aos
+```lua
+local json = require("json")
+
+_0RBIT = "BaMK1dfayo75s3q1ow6AO64UDpD9SEFbeE8xYrY2fyQ"
+_0RBT_TOKEN = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
```
Comment (megabyte0x) [inline src/guides/0rbit/get-request.md:38]: @Not-Sarthak please change this to `_0RBT_POINTS`

**Comment by PSkinnerTech** [comment]
LGTM!

**Pull request merged**